.travis.yml

```diff
@@ -0,0 +1,51 @@
+sudo: 'required'
+
+services:
+- 'docker'
+
+env:
+  global:
+    - DOCKER_TAG=0.1.${TRAVIS_BUILD_NUMBER}
+    - DOCKER_PUSH=1 # if DOCKER_PUSH is defined, `make docker-*` uploads images to dockerhub.
+  matrix:
+    # - TARGETS="docker-builder" # currently takes ~ 2h (needs to compile GHC) which is more than travis permits. This image will, until a more recent ghc is available as an alpine package, be uploaded manually/through wire-internal CI"
+    - TARGETS="docker-deps"
+    - TARGETS="docker-intermediate docker-migrations"
+    - TARGETS="docker-intermediate
+               docker-exe-api-loadtest
+               docker-exe-api-smoketest
+               docker-exe-bonanza
+               docker-exe-brig
+               docker-exe-brig-index
+               docker-exe-brig-integration
+               docker-exe-brig-schema
+               docker-exe-cannon
+               docker-exe-cargohold
+               docker-exe-cargohold-integration
+               docker-exe-galley
+               docker-exe-galley-integration
+               docker-exe-galley-journaler
+               docker-exe-galley-schema
+               docker-exe-gundeck
+               docker-exe-gundeck-integration
+               docker-exe-gundeck-schema
+               docker-exe-kibana-raw
+               docker-exe-kibanana
+               docker-exe-makedeb
+               docker-exe-metrics-collector
+               docker-exe-proxy
+               docker-exe-ropes-aws-auth-test
+               docker-exe-ropes-aws-test
+               docker-exe-zauth
+              "
+
+script:
+  - docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD}
+  - make ${TARGETS[@]}
+
+notifications:
+  email: false
+
+branches:
+  only:
+    - develop
```

Comment:
This YAML is not valid.. (notice the `script`, `notification`, etc are not green as `service`, `env` keys.. 

https://codebeautify.org/yaml-validator/cb9d3cdc

Reply:
Travis seems to parse it fine, it builds the intended things, and both https://yamlvalidator.com/ and http://www.yamllint.com/ say "valid yaml".

I didn't try what happens if there's a 

```yaml
env:
    matrix:
        - >
          TARGETS="..."
```
bit, which might make your quoted validator happy but might confuse travis, I'm not sure.

Reply:
@jschaul I think is not valid as on travis it [shows two targets](https://travis-ci.org/wireapp/wire-server/builds/349934386?utm_source=github_status&utm_medium=notification) while you have three. Maybe close the `"` on the same line instead of a separate (Line 39)

Reply:
the build you quoted (84) was a manual test with a different config, trying the yaml multi-line string syntax ">", which failed. 

The travis file as-is on the develop branch is valid for travis' purposes, builds three targets, see e.g. [build 91](https://travis-ci.org/wireapp/wire-server/builds/350267424)

I can put all the targets space-separated into the same line, though that will make one long, unreadable line. Would this help you in your environment?

Reply:
I'm just trying to figure out how can I run my stuff in #208 and I can't find out a way :)